### ISSUE TITLE
Carriage return is a one way road

### ISSUE THREAD

**Issue by allan2** [tetchel/Shexter#4]
Cannot go back to previous line in send message field

**Comment by tetchel** [comment]
good point hadn't thought of that. who puts newlines in their texts anyway

**Comment by allan2** [comment]
Sometimes I like to send

TORONTO
ORONTO
RONTO
ONTO
NTO
TO
O

after a big sporting event win, e.g. Leafs winning Stanley Cup

**Comment by tetchel** [comment]
~~shift-enter can be implemented it seems
http://stackoverflow.com/questions/4581772/using-pyhook-to-respond-to-key-combination-not-just-single-keystrokes~~

darn that's windows only

**Comment by tetchel** [comment]
now that i use this regularly this bug is driving me crazy
